eks-hybrid-iam-ra-setup.sh

```diff
@@ -3,14 +3,19 @@
 exec >&2
 set -eu -o pipefail
 
-declare -r SECRETS_DIR="/.bottlerocket/rootfs/root/.aws"
+declare -r HOST_ROOTFS="/.bottlerocket/rootfs"
+declare -r SECRETS_DIR="${HOST_ROOTFS}/root/.aws"
+declare -r AWS_SIGNING_HELPER_UPDATE_SYSTEMD_SERVICE=aws_signing_helper_update.service
+declare -r AWS_SIGNING_HELPER_UPDATE_SYSTEMD_SERVICE_DESTINATION="${HOST_ROOTFS}/run/systemd/system/${AWS_SIGNING_HELPER_UPDATE_SYSTEMD_SERVICE}"
 
 DRY_RUN="false"
+ENABLE_CREDENTIALS_FILE="false"
 for opt in "$@"; do
     optarg="$(expr "${opt}" : '[^ =]*[= ]\(.*\)')"
     case "${opt}" in
         --certificate=*) NODE_CERT_DATA="${optarg}" ;;
         --key=*) NODE_KEY_DATA="${optarg}" ;;
+        --enable-credentials-file=*) ENABLE_CREDENTIALS_FILE="${optarg}" ;;
```

Comment:
Seems like this should be mandatory?
```suggestion
        --credentials-file=*) CREDENTIALS_FILE="${optarg}" ;;
```

Reply:
The arg `enable-credentials-file` comes from the [nodeadm config flag of the same name](https://docs.aws.amazon.com/eks/latest/userguide/hybrid-nodes-add-ons.html#hybrid-nodes-add-ons-pod-id) which enables users to use the Pod Identity agent on hybrid nodes. On the `nodeadm` side, this flag when set to `true` will configure `nodeadm` to create a credentials file to be configured on the node under `/eks-hybrid/.aws/credentials`, which will be used by `eks-pod-identity-agent` pods. So this flag is just a bool that indicates whether to setup the pod identity stuff or not.